ordering/templates/index.html

```diff
@@ -7,225 +7,207 @@
-    <link rel="stylesheet"
-          href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-          integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
-          crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css">
     <link rel="stylesheet" href="{{ url_for('static', filename='css/index.css') }}">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css" />
 </head>
 
 <body>
-<main class="container">
-    <div class="row">
-        <div class="col">
-            <h1>Arrowverse Episode Order</h1>
-        </div>
-        <div class="col-2">
-            <a href="https://github.com/AceFire6/ordered-arrowverse"
-               class="btn btn-outline-secondary"
-               target="_blank">
-                Project on Github
-            </a>
-        </div>
-        <div>
-            <a href="{{ url_for('recent_episodes', hide_show=hidden_show_list) }}"
-               class="btn btn-outline-secondary"
-               target="_blank">
-                RSS Feed
-                <img src="https://www.mozilla.org/media/img/trademarks/feed-icon-14x14.2168a573d0d4.png" rel="icon">
-            </a>
-        </div>
-    </div>
-    <hr>
-    <div class="row">
-        <div class="col">
-            <div class="row">
-
-                <div class="col-11">
-                    <form action="{{ url_for('index') }}" id="filter-form" method="get" class="row form-inline">
-                        <div class="col-7">
-                            <input name="newest_first" value="{{ newest_first }}" title="Newest First" type="hidden">
-                            <select id="show-filter-select"
-                                title="Filter options"
-                                name="hide_show"
-                                multiple="multiple"
-                                class="form-control" >
-                                {% for show_id, show_data in show_list.items() %}
-                                    <option value="{{ show_id }}" {% if show_id in hidden_show_list %}selected{% endif %}>
-                                        {{ show_data.name }}
-                                    </option>
-                                {% endfor %}
-                            </select>
-                        </div>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <div class="container">
+            <span class="navbar-brand">Arrowverse Episode Order</span>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
```

Comment:
I'd prefer this to be wrapped so it's easier to see all the attributes on the element